File: tzapi4android/src/main/java/com/tianzunchina/android/api/control/TZApplication.java
package com.tianzunchina.android.api.control;

import android.app.Application;

/**
 * CraetTime 2016-3-14
 * @author SunLiang
 */
public class TZApplication extends Application {
    private static TZApplication instence;

    public static TZApplication getInstence(){
        return instence;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        instence = this;
    }
}

File: tzapi4android/src/main/java/com/tianzunchina/android/api/control/TZApplication.kt
package com.tianzunchina.android.api.control

import android.app.Application

/**
 * Created by admin on 2016/3/4 0004.
 */
class TZApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        instance = this
    }

    companion object {
        var instance: TZApplication? = null
            private set
    }
}